{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorServerException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorServerException","preciseIdentifier":"s:10AWSAppflow24ConnectorServerExceptionV","text":"ConnectorServerException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappflow\/createflowoutputerror\/connectorserverexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/CreateFlowOutputError\/connectorServerException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorServerException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorServerException","preciseIdentifier":"s:10AWSAppflow24ConnectorServerExceptionV"},{"kind":"text","text":")"}],"title":"CreateFlowOutputError.connectorServerException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10AWSAppflow21CreateFlowOutputErrorO24connectorServerExceptionyAcA09ConnectorgH0VcACmF","modules":[{"name":"AWSAppflow"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppflow\/documentation\/AWSAppflow","doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/CreateFlowOutputError"]]},"references":{"doc://AWSAppflow/documentation/AWSAppflow/CreateFlowOutputError":{"role":"symbol","title":"CreateFlowOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"CreateFlowOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/CreateFlowOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"CreateFlowOutputError"}],"url":"\/documentation\/awsappflow\/createflowoutputerror"},"doc://AWSAppflow/documentation/AWSAppflow/CreateFlowOutputError/connectorServerException(_:)":{"role":"symbol","title":"CreateFlowOutputError.connectorServerException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectorServerException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"ConnectorServerException","preciseIdentifier":"s:10AWSAppflow24ConnectorServerExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/CreateFlowOutputError\/connectorServerException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow\/createflowoutputerror\/connectorserverexception(_:)"},"doc://AWSAppflow/documentation/AWSAppflow":{"role":"collection","title":"AWSAppflow","abstract":[],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow","kind":"symbol","type":"topic","url":"\/documentation\/awsappflow"},"doc://AWSAppflow/documentation/AWSAppflow/ConnectorServerException":{"role":"symbol","title":"ConnectorServerException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConnectorServerException"}],"abstract":[{"type":"text","text":"An error occurred when retrieving data from the connector endpoint."}],"identifier":"doc:\/\/AWSAppflow\/documentation\/AWSAppflow\/ConnectorServerException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectorServerException"}],"url":"\/documentation\/awsappflow\/connectorserverexception"}}}